staging/src/k8s.io/apiserver/plugin/pkg/audit/dynamic/dynamic_test.go

```diff
@@ -176,6 +177,24 @@ func TestDynamic(t *testing.T) {
 		require.NoError(t, err, "unable to find events sent to sink 1")
 		err = checkForEvent(eventList2, testEvent)
 		require.NoError(t, err, "unable to find events sent to sink 2")
+
+		// test that backends don't race
```

Comment:
Is the idea that the race detector should be triggered if there is a race here? Is there any additional verification that can be done?

Reply:
yep thats the idea. I added some checks that test that we aren't missing events and that the events received are all consistent. Let me know what you think